src/dualize.jl

```diff
@@ -1,19 +1,22 @@
 export dualize
 
 # MOI dualize
-function dualize(primal_model::MOI.ModelLike; dual_names::DualNames = DualNames("", ""))
+function dualize(primal_model::MOI.ModelLike; dual_names::DualNames = DualNames(),
```

Comment:
What are `variable_parameters` and `ignore_objective` ? Shouldn't this function have a docstring ?

Reply:
Yes, it needs docstrings and tests.
I might have better names, I used those to get things up and running.
One of the methods to solve bilevel problems is to add the KKT conditions of the lower level problem to the upper level problem.
However, these KKT conditions are somewhat special, the variables coming from the upper level are "parameters" for the lower level, so they are considered constants during the dualization process.
In the KKT conditions (for LPs) the objective function is implicitly considered in the RHS of the dual problem.
I am using those functions in: https://github.com/joaquimg/BilevelJuMP.jl
In particular: https://github.com/joaquimg/BilevelJuMP.jl/blob/d1a7cbf520c4db730d3cf6b1444e664c7331563f/src/moi.jl#L41

Reply:
We should add a reference to BilevelJuMP in the README as people might come to Dualization because they need to solve bilinear optimization problems and they would be interested to read that there is a JuMP extension for this

Reply:
Excelent idea, I will add comenta to the source and info in readme